src/libs/ascent/runtimes/flow_filters/ascent_runtime_rendering_filters.cpp

```diff
@@ -200,6 +189,44 @@ check_renders_surprises(const conduit::Node &renders_node)
 
   std::vector<std::string> r_ignore_paths;
   r_ignore_paths.push_back("phi_theta_positions");
+  r_ignore_paths.push_back("camera");
+
+  // Valid Ascent input camera format
+  std::vector<std::string> c_ascent_valid_paths;
+  c_ascent_valid_paths.push_back("2d");
+  c_ascent_valid_paths.push_back("look_at");
+  c_ascent_valid_paths.push_back("position");
+  c_ascent_valid_paths.push_back("up");
+  c_ascent_valid_paths.push_back("fov");
+  c_ascent_valid_paths.push_back("xpan");
+  c_ascent_valid_paths.push_back("ypan");
+  c_ascent_valid_paths.push_back("zoom");
+  c_ascent_valid_paths.push_back("near_plane");
+  c_ascent_valid_paths.push_back("far_plane");
+  c_ascent_valid_paths.push_back("azimuth");
+  c_ascent_valid_paths.push_back("elevation");
+
+  // Valid Visit input camera format
+  std::vector<std::string> c_visit_valid_paths;
+  c_visit_valid_paths.push_back("view_normal");
+  c_visit_valid_paths.push_back("focus");
+  c_visit_valid_paths.push_back("view_up");
+  c_visit_valid_paths.push_back("view_angle");
+  c_visit_valid_paths.push_back("parallel_scale");
+  c_visit_valid_paths.push_back("near_plane");
+  c_visit_valid_paths.push_back("far_plane");
+  c_visit_valid_paths.push_back("image_pan");
+  c_visit_valid_paths.push_back("image_zoom");
+  c_visit_valid_paths.push_back("perspective");
+  c_visit_valid_paths.push_back("eye_angle");
+  c_visit_valid_paths.push_back("center_of_rotation_set");
+  c_visit_valid_paths.push_back("center_of_rotation");
+  c_visit_valid_paths.push_back("axis_3d_scale_flag");
+  c_visit_valid_paths.push_back("axis_3d_scale");
+  c_visit_valid_paths.push_back("shear");
+  c_visit_valid_paths.push_back("window_valid");
```

Comment:
I guess this is the style everywhere? It might be worthwhile to consider using initializer lists for this.